### ISSUE TITLE
ControlStructureSpacingSniff doesn't consider commented out lines

### ISSUE THREAD

**Issue by lcobucci** [slevomat/coding-standard#677]
PHP | squizlabs/php_codesniffer | slevomat/coding-standard 
-----| ---------------------- | -----------------------------
7.2.17 | 3.4.2 | 5.0.4

`ControlStructureSpacingSniff` detects that there shouldn't be any line before of the `return` statement of this snippet and fails to fix it:

```php
<?php

function compare(object $one, object $other): bool
{
    // Weak comparison in PHP compares both the state and class,
    // so two subclasses of with the same state
    // won't match up just because they have the values.
    /** @noinspection TypeUnsafeComparisonInspection,PhpNonStrictObjectEqualityInspection */
    return $one == $other;
}
```

```
FILE: aa.php
------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
FOUND 1 ERROR AFFECTING 1 LINE
------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
 9 | ERROR | [x] Expected 0 lines before "return", found 3.
   |       |     (SlevomatCodingStandard.ControlStructures.ControlStructureSpacing.IncorrectLinesCountBeforeFirstControlStructure)
------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
PHPCBF CAN FIX THE 1 MARKED SNIFF VIOLATIONS AUTOMATICALLY
------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
```

```
PHPCBF RESULT SUMMARY
----------------------------------------------------------------------
FILE                                                  FIXED  REMAINING
----------------------------------------------------------------------
aa.php                                                FAILED TO FIX
----------------------------------------------------------------------
A TOTAL OF 1 ERROR WERE FIXED IN 1 FILE
----------------------------------------------------------------------
PHPCBF FAILED TO FIX 1 FILE
----------------------------------------------------------------------

Time: 214ms; Memory: 4MB
```

I'd expect this occurrence to be ignored by the sniff, since having explanation before of the `return` statement is somewhat acceptable :laughing:.

**Comment by kukulich** [comment]
@lcobucci could you try to merge both comments to one?

**Comment by lcobucci** [comment]
@kukulich that does make it work and it's an okay solution for my case. Thanks!
